MaxText/layers/attention_op.py

```diff
@@ -677,6 +678,7 @@ def apply_attention(
       use_ragged_attention: bool = False,
       previous_chunk: Any = None,
       bidirectional_mask: Any = None,
+      sinks: Array = None,
```

Comment:
could you error out if sinks is provided, but there is no implementation, e.g. for cudnn_flash

Reply:
Sure! Will raise the error accordingly.